fbw-a32nx/src/systems/instruments/src/MsfsAvionicsCommon/displayUnit.tsx

```diff
@@ -53,6 +63,21 @@ export class DisplayUnit extends DisplayComponent<DisplayUnitProps> {
             if (!document.documentElement.classList.contains('animationsEnabled')) {
                 document.documentElement.classList.add('animationsEnabled');
             }
+
+            const dmcKnob = SimVar.GetSimVarValue('L:A32NX_EIS_DMC_SWITCHING_KNOB', 'Enum');
```

Comment:
Couldn't subscribe to changes on A32NX_EIS_DMC_SWITCHING_KNOB, so I had to resort to this solution.